src/v2/Apps/Purchase/routes.tsx

```diff
@@ -18,11 +18,6 @@ export const routes: RouteConfig[] = [
         }
       }
     `,
-    prepareVariables: (params, props) => {
```

Comment:
This was not doing anything. The variables were coming from the `defaultValue` set in the fragment so removed it from here.